plugins/destination/postgresql/client/client_test.go

```diff
@@ -4,14 +4,14 @@ import (
 	"os"
 	"testing"
 
-	"github.com/cloudquery/plugin-sdk/plugins/destination"
-	"github.com/cloudquery/plugin-sdk/specs"
+	"github.com/cloudquery/plugin-sdk/v2/plugins/destination"
+	"github.com/cloudquery/plugin-sdk/v2/specs"
 )
 
 func getTestConnection() string {
 	testConn := os.Getenv("CQ_DEST_PG_TEST_CONN")
 	if testConn == "" {
-		return "postgresql://postgres:pass@localhost:5432/postgres?sslmode=disable"
+		return "postgresql://root@localhost:26257/postgres?sslmode=disable"
```

Comment:
```suggestion
		return "postgresql://postgres:pass@localhost:5432/postgres?sslmode=disable"
```
I think this wasn't intended